### DISCUSSION TITLE
Profiles and saved game screen

### DISCUSSION THREAD

**Discussion by hierophantom** [Keriew/augustus#387] [Ideas]
Hi, My suggestion is to improve the main menu UX by adding the option to have "Profiles" that saves your progress and previously played missions.
I've made a very basic video that shows the possible scheme to such a system.
https://user-images.githubusercontent.com/11699185/118677925-4d876c00-b805-11eb-920e-d7b344650968.mov

**Every profile shows:**
- Players name
- Last save date
- Current mission
- Current Rank

**Every profile page shows:**
- Current mission
- Ability to go to previous missions

Hope it makes sense!

**Comment by crudelios** [comment]
It does make sense and in fact it's a planned feature which I will eventually get to 👍

**Reply by hierophantom** [reply]
Great! if you need someone to do the graphics of it i'll be happy to!
